### DISCUSSION TITLE
Backup and restore configuration of streams on the new server.

### DISCUSSION THREAD

**Discussion by camtour** [ant-media/Ant-Media-Server#6185] [Q&A] [answered]
Hi, 
I have 70 ipcamera streams for an application. 
Is there any guide about how to backup the configuration from an existing server and restore it on a new server?

**Comment by muratugureminoglu** [comment, marked as answer]
Hi @camtour 

1. Complete the Ant Media installation on the new server and change the directory.
```
mv /usr/local/antmedia{,_bck}
```
2. Then upload the /usr/local/antmedia directory from the old server to the new server and copy it to the /usr/local/ directory.

3. Fix the permissions of the files you copied.
```
chown -R antmedia:antmedia /usr/local/antmedia/
```
4. Restart Ant Media Server.
```
systemctl restart antmedia
```
That's it. Please let me know if you face any issues.

Regards.

**Reply by camtour** [reply]
Thank you for your kind response. 
I think the permission modification part should be changed to
 chown -R antmedia:antmedia /usr/local/antmedia/

**Reply by muratugureminoglu** [reply]
Yes you are right, My bad.
